jsk_fetch_robot/jsk_fetch_diagnosis/scripts/check_driver_boards.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import argparse
+import os
+import sys
+
+import subprocess
+import rospy
+from jsk_fetch_diagnosis.msg import *
```

Comment:
Do not use `import *`, which may cause unexpected import.

Also it is better to import alphabetically and  put space before `jsk_fetch_diagnosis`

```
import argparse
import os
import subprocess
import sys

import rospy

from jsk_fetch_diagnosis.msg import BoardInfo
```

Reply:
changed import order